### ISSUE TITLE
How can I get loss when training

### ISSUE THREAD

**Issue by RDADADA** [xuhongzuo/DeepOD#59]
Is there any way to get training loss during training?
It’s printed but can I get it?

epoch  1, training loss: 0.998026, time: 3.3s   <- training loss

**Comment by xuhongzuo** [comment]
Hi @RDADADA! 
The loss value you see printed comes from a variable in the training loop. You can directly use that variable.
